scipy/optimize/_optimize.py

```diff
@@ -2248,12 +2248,14 @@ def get_bracket_info(self):
             if (xa > xc):  # swap so xa < xc can be assumed
                 xc, xa = xa, xc
             if not ((xa < xb) and (xb < xc)):
-                raise ValueError("Not a bracketing interval.")
+                raise ValueError("Bracketing values (xa, xb, xc) do not\
+                 fulfill this requirement: (xa < xb) and (xb < xc)")
```

Comment:
This is how these should be formatted.
```suggestion
                raise ValueError(
                    "Bracketing values (xa, xb, xc) do not"
                    " fulfill this requirement: (xa < xb) and (xb < xc)"
                )
```